odk1-src/form-functions.rst

```diff
@@ -0,0 +1,502 @@
+  
+  .. seealso:: :ref:`The complete example of this form, with a video. <complex-conditional-example>`
+  
+  
+.. function:: selected-at(multi_select_question, n)
+
+  Returns the :th:`name` of the :arg:`n`\th selected choice of the :arg:`multi_select_question`. (Selected choices are zero-indexed.)
+  
+  .. note::
+  
+    This function returns the :th:`name`, not the :th:`label`,
+    of the selected choice.
+    To get the label in the current language,
+    use :func:`jr:choice-name`.
+  
+  Returns an empty string if the index does not exist.
+  
+  .. image:: /img/form-functions/selected-at-0.* 
+    :alt:
+  
+  .. image:: /img/form-functions/selected-at-1.* 
+    :alt:
+  
+  .. rubric:: XLSForm
+  
+  .. csv-table:: survey
+    :header: type, name, label, hint, calculation
+  
+    select_multiple colors, color_prefs, What colors do you like?, Select three.
+    calculate, color_0, , ,"selected-at(${color_prefs}, 0)"
+    calculate, color_1, , ,"selected-at(${color_prefs}, 1)"
+    calculate, color_2, , ,"selected-at(${color_prefs}, 2)"
+    note, color_note, Selected colors:, ${color_0} <br> ${color_1} <br> ${color_2}
+    
+    
+
+.. function:: indexed-repeat(name, group, i [, sub_grp, sub_i [, sub_sub_grp, sub_sub_i ]])
+
+  Returns the response value of question ``name``
+  from the repeat-group ``group``,
+  in iteration ``i``.
+  
+  Nested repeat groups can be accessed 
+  using the ``sub`` and ``sub_sub`` parameters,
+  which can be extended to three levels of nesting.
+
+  
+Counting responses
+~~~~~~~~~~~~~~~~~~~
+
+.. function:: count-selected(multi_select_question)
+
+  Returns the number of choices selected in ``multi_select_question``.
+ 
+
+.. function:: count(nodeset)
+
+  Returns the number of items in ``nodeset``. This can be used to count the number of repetitions in a :ref:`repeat group <repeats>`.
+
+.. function:: count-non-empty(nodeset)
+
+  Returns the number of non-empty members of ``nodeset``.
+
+ 
+Doing math with responses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+.. function:: sum(nodeset)
+
+  Returns the sum of the members of ``nodeset``.
+
+.. function:: max(nodeset)
+
+  Returns the largest member of ``nodeset``.
+
+.. function:: min(nodeset)
+
+  Returns the smallest memberd of ``nodeset``.
+
+
+Accessing information about the form
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+ 
+.. function:: jr:choice-name(select_question, choice_name)
+
+  Returns the label value, in the active language, associated with the ``choice_name`` in the list of choices for the ``select_question``.
+
+.. function:: jr:itext(name) 
+
+  Returns the itext in the active language for the named item.
+  
+.. function:: position(xpath)
+
+  Returns an integer equal to the 1-indexed position of the current node
+  within the node defined by ``xpath``.
+  
+  Most often this is used in the form ``position(..)``
+  to identify the current iteration index.  
+  
+  
+String handling
```

Comment:
Might be OK to just call this string. Since you don't call it "math and numbers handling"